app/client/src/components/pages/Community.Tabs.DrinkingWater.js

```diff
@@ -640,26 +681,65 @@ function DrinkingWater() {
                       )}
 
                       {providers.length > 0 && (
-                        <AccordionList
-                          title={
-                            <>
-                              There {providers.length === 1 ? 'is' : 'are'}{' '}
-                              <strong>{providers.length}</strong> public water{' '}
-                              {providers.length === 1 ? 'system' : 'systems'}{' '}
-                              serving <em>{county}</em> county.
-                            </>
-                          }
-                          onSortChange={(sortBy) =>
-                            setProvidersSortBy(sortBy.value)
-                          }
-                          sortOptions={providerSorts}
-                        >
-                          {sortWaterSystems(
-                            providers,
-                            providersSortBy,
-                            false,
-                          ).map((item) => createAccordionItem(services, item))}
-                        </AccordionList>
+                        <>
+                          {tribalProviderCount > 0 && (
```

Comment:
I think the Tribal Status switches should always be visible and just be disabled if the `tribalProviderCount` equals 0. This is what we do for the Ground Water and Surface Water switches.